File: LeetCode/Simulation/JS/Leetcode1754-largest-merge-of-two-strings.js
var minMovesToSeat = function(seats, students) {
    seats.sort((a, b) => a - b);
    students.sort((a, b) => a - b);
    let ans = 0;
    for (let i = 0;i < students.length;i++) {
        ans += Math.abs(students[i] - seats[i]);
    }
    return ans;
};

File: LeetCode/Simulation/Python/Leetcode1754-largest-merge-of-two-strings.py
class Solution:
    def minMovesToSeat(self, seats: List[int], students: List[int]) -> int:
        students.sort()
        seats.sort()
        ans = 0 
        for i in range(len(seats)):
            ans += abs(students[i] - seats[i])
        return ans
